tck/src/main/java/ee/jakarta/tck/data/common/cdi/Person.java

```diff
@@ -17,11 +17,7 @@
 
 /**
  * A test entity that will be persisted to a repository.
- * Uses the custom {@code @PersonEntity} annotation.
- * 
- * @see ee.jakarta.tck.data.common.cdi.PersonEntity
  */
-@PersonEntity
 public class Person {
```

Comment:
Should we put the NoSQL and/or Persitence `@Entity` onto this to help test that Jakarta Data providers against which the TCK is run do not try to handle it because its repository specifically asks for the TCK mock Data provider `provider = Directory.PERSON_PROVIDER`?